release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+
+[id="ocp-4-2-full-stack-automated-deployments"]
+==== Full stack automated deployments
+
+In {product-title} {product-version}, there is expanded support for full stack
+automated deployments to include AWS, Microsoft Azure, Google Cloud Platform (GCP),
+and Red Hat OpenStack Platform, as well as adding GCP to the existing list of
+user provisioned infrastructure supported providers that already includes AWS,
+Bare Metal, and VMware vSphere.
+
+[id="ocp-4-2-operators"]
+=== Operators
+
+[id="ocp-4-2-operator-installation"]
+=== Operator installation
+
+Previously, only users carrying `cluster-admin` roles  were allowed to install
+Operators. In {product-title} 4.2, the `cluster-admin` can select namespaces in
+which namespace administrators can install Operators self-sufficiently. The
+`cluster-admin` defines the ServiceAccount in this namespace; all installed
+Operators in this namespace get equal or lower permissions of this
+ServiceAccount.
+
+[id="ocp-4-2-machine-config-operator"]
+==== Machine Config Operator
+
+The Machine Config Operator (MCO) provides cluster-level configuration, enables
+rolling upgrades, and prevents drift between new and existing nodes.
+
+[id="ocp-4-2-node-feature-discovery-operator"]
+==== Node Feature Discovery Operator
+
+The Node Feature Discovery (NFD) Operator detects hardware features available
+on each node and advertises those features using node labels.
+
+Features
+CPU features managed by NFD include:
+* cpuid
+* hardware_multithreading
+* power
+* pstate
+
+Kernel features managed by NFD include:
+* config
+* selinux_enabled
+* version
+* os_version
+
+Other features managed by NFD include:
+
+* NVMe
+* NUMA
+* SR-IOV
+* GPUs
+
+The NFD Operator manages the installation and lifecycle of the NFD DaemonSet.
+Access the NFD Operator in OperatorHub.
+
+[id="ocp-4-2-node-tuning-operator-enhancements"]
+==== Node Tuning Operator enhancements
+
+The Node Turning Operator was first introduced in {product-title} 4.1 and
+manages cluster node-level tuning; The default CR is meant for delivering
+standard node-level tuning. The enhancements in {product-title} 4.2 allow for
+customizing the tunings for things such as high performance.
+
+For custom tuning, create your own tuned custom resources (CRs). Newly created
+CRs will be combined with the default CR and custom tuning applied to nodes
+based on node or pod labels and profile priorities.
+
+[id="ocp-4-2-special-resources-operator"]
+==== Special Resources Operator
+
+The Special Resources Operator (SRO) is an operator for orchestrating special
+resources in a cluster that need special handling. SRO implements a simple state machine, where each state implements a state and has a
+validation step.
+
+{product-title} introduces reference implementation of the SRO for NVIDIA GPUs.
+The SRO deploys:
+
+* NVIDIA Cuda driver container
+* NVIDIA GPU device plug-in daemonset
+* NVIDIA DGCM_exporter for GPU metrics
+
```

Comment:
@zvonkok @ahardin-rh should SRO documentation be included in product docs? It is a community operator at this point, right?

Reply:
@ashishkamra I will pull it out for now. Thanks!